model/clusters_mgmt/v1/gcp_inquiries_resource.model

```diff
@@ -32,5 +32,9 @@ resource GCPInquiries {
     locator Regions {
         target AvailableRegionsInquiry
     }
+    // Reference to the resource that manages gcp machine types by regions.
+    locator MachineTypes {
+        target GCPRegionMachineTypesInquiry
```

Comment:
There exists `AWSRegionMachineTypesInquiry`, I'm not sure if I should just reuse it or if I should add new model as in the PR.

Reply:
I'm good with adding a new model as you did here.